### DISCUSSION TITLE
Outputing mono to a stereo output

### DISCUSSION THREAD

**Discussion by olup** [pschatzmann/arduino-audio-tools#163] [Q&A] [answered]
This is just a general question :  how to go with getting mono output on a stereo i2s board ?

I think the idea would be to merge right and left channels before feeding it to the i2s ? Or it can be done after the i2s output but will require a couple of resistor and some wiring.

**Comment by pschatzmann** [comment, marked as answer]
To my knowlege, the I2S protocol is basically a stereo protocol. 

You are right, if you have a stereo signal and want to use only one signal you need to combine both channels. In my library this is automatically done when you set cfg.channels = 1; 

You then just can use either of the two I2S pins. They have the same signal.

**Reply by olup** [reply]
Hey, thanks for this quick answer. Now in the context of an Audiokit board, where I don't control the wiring, is there a way to programmatically merge left and right channel and then feed them to the stereo output so that I can wire a single speaker to, say, the right channel, and be sure to get a mono representation of the track (and not just the right channel) ?

**Reply by olup** [reply]
Sorry, you just answered exactly that ! thanks again.
